rust/ql/src/queries/security/CWE-312/CleartextLogging.ql

```diff
@@ -17,7 +17,7 @@ import rust
 import codeql.rust.security.CleartextLoggingExtensions
 import codeql.rust.dataflow.DataFlow
 import codeql.rust.dataflow.TaintTracking
-import codeql.rust.dataflow.internal.DataFlowImpl
+import codeql.rust.dataflow.internal.Content
```

Comment:
I think it would be better if we expose the relevant `Content` sub classes as aliases inside `DataFlow.qll` (excluding `FunctionCallArgumentContent` and `FunctionCallReturnContent`).